SupyML/plugin.py

```diff
@@ -99,7 +101,8 @@ def __init__(self, plugin, irc, msg, code, maxNodes):
         self.warnings = []
         self._maxNodes = maxNodes
         self.nodesCount = 0
-        self.data = self._parse(code)
+        self.rawData = None
```

Comment:
is this used?

Reply:
in line 262 in def eval()...
if parser.rawData is not Null:
rawData is normally set to the executed irc message if there is one, but wasn't initialized in the constructior.
In some error cases it was never set. That led to an uncaught exception (accessing nonexisting attribute)
I think this was a hack to allow non-privmessage commands like op or voice run in eval, but it would not have worked nested and this is now done another way. It might make sense to get rid of it altogether. I think I'll do that.

Reply:
no longer used in latest version